### PR TITLE
add hard JOA checklist

### PR DESCRIPTION
**Pull request by wslack**
We need to make a final list of things to check before any JOA goes live

### CODE REVIEW

**Comment by wslack** [comment]
we could also add: 

- Have you confirmed the total number of applicants is workable given your SME availability? (taking and parentheticals or roles into account?)
- Do you have HR time set aside to process the applications after the JOA closes (either on time or early?)
- Have you set up training for your SMEs so they can start resume review as soon as the posting closes?

File: pages/phases/job-announcement/index.html [hunk 1]
```diff
@@ -62,6 +62,18 @@ <h4 class="chp-highlight__title">
...
+<h2>Final checklist before JOA Posting</h2>
+
+<ul>
+<li>If using USA Staffing, have you set up the manual assessments in order to import the SMEQA assessment results into Staffing? <b>Not doing this will cause a large amount of manual work.</b></li>
+<li>Do you have a final private copy of the competencies and proficiencies that SMEs have approved with language matching the JOA?</li>
+<li>Do you have language calling out the page limit for resume review in the JOA</li>
+<li>Have you included language allowing candidates to be considered for other hiring actions?</li>
+<li>Have you included the opt-in questionnaire item on being considered for employment at other agencies?</li>
+<li>Do you have a link to the questionnaire in your JOA?</li>
+<li>Are the SMEs and HMs? ready to do recruitment when you send them the link?</li>
```
Comment (stephaniefaith) [inline pages/phases/job-announcement/index.html:74]:
```suggestion
<li>Are the SMEs and HMs ready to do recruitment when you send them the link?</li>
```

File: pages/phases/job-announcement/index.html [hunk 2]
```diff
@@ -62,6 +62,18 @@ <h4 class="chp-highlight__title">
...
 
+<h2>Final checklist before JOA Posting</h2>
+
+<ul>
+<li>If using USA Staffing, have you set up the manual assessments in order to import the SMEQA assessment results into Staffing? <b>Not doing this will cause a large amount of manual work.</b></li>
+<li>Do you have a final private copy of the competencies and proficiencies that SMEs have approved with language matching the JOA?</li>
+<li>Do you have language calling out the page limit for resume review in the JOA</li>
+<li>Have you included language allowing candidates to be considered for other hiring actions?</li>
+<li>Have you included the opt-in questionnaire item on being considered for employment at other agencies?</li>
+<li>Do you have a link to the questionnaire in your JOA?</li>
```
Comment (stephaniefaith) [inline pages/phases/job-announcement/index.html:73]:
```suggestion
<li>Do you have a link to the questionnaire in your JOA?</li>
<li>Did you consider including language telling applicants the announcement will remain open for either 240 days or 365 days to maximize selections off the certificate throughout the year?</li>
```

**Comment by stephaniefaith** [comment]
@wslack  can you add this too:

<li>Did you consider including language telling applicants the announcement will remain open for either 240 days or 365 days to maximize selections off the certificate throughout the year?</li>

**Review by stephaniefaith — APPROVED** [review]

**Comment by wslack** [comment]
> Did you consider including language telling applicants the announcement will remain open for either 240 days or 365 days to maximize selections off the certificate throughout the year?

Is this explained elsewhere on the site yet?

**Review by amy-usds — APPROVED** [review]

**Pull request merged**